technische-universitat-dresden-historische-musikwissenschaft-note.csl

```diff
@@ -464,7 +466,7 @@
             <text macro="contributors-long"/>
             <text macro="title-long"/>
             <group>
-              <text macro="container"/>
+              <text macro="container" suffix=", "/>
```

Comment:
Probably better as a delimiter in the group.

Reply:
You are right